lms/djangoapps/verify_student/models.py

```diff
@@ -1103,13 +1104,13 @@ def invalidate_deadline_caches(sender, **kwargs):  # pylint: disable=unused-argu
     cache.delete(VerificationDeadline.ALL_DEADLINES_CACHE_KEY)
 
 
-class VerificationCheckpoint(models.Model):
-    """Represents a point at which a user is asked to re-verify his/her
-    identity.
-
-    Each checkpoint is uniquely identified by a
-    (course_id, checkpoint_location) tuple.
+class VerificationCheckpoint(DeprecatedModel):  # pylint: disable=model-missing-unicode
+    """
+    DEPRECATED - do not use. To be removed in next openedx release.
```

Comment:
s/openedx/Open edX/

Also, it isn't being removed in the next, it's the one after the next.  Do we want to say Hawthorn here? Or, "in a future Open edX release"?